airflow/cli/commands/role_command.py

```diff
@@ -42,3 +44,43 @@ def roles_create(args):
     for role_name in args.role:
         appbuilder.sm.add_role(role_name)
     print(f"Added {len(args.role)} role(s)")
+
+
+@suppress_logs_and_warning
+def roles_export(args):
+    """
+    Exports all the rules from the data base to a file.
+    """
+    from airflow.www.security import EXISTING_ROLES
+    appbuilder = cached_app().appbuilder
+    roles = appbuilder.sm.get_all_roles()
+    exporting_roles = [role.name for role in roles if role.name not in EXISTING_ROLES]
+    with open(os.path.expanduser(args.export), 'w') as file:
+        file.write(json.dumps(exporting_roles, sort_keys=True, indent=4))
+        print(f"{len(exporting_roles)} roles successfully exported to {file.name}")
```

Comment:
```suggestion
    filename = os.path.expanduser(args.export)
    with open(filename, 'w') as f:
        json.dumps(f, exporting_roles, sort_keys=True, indent=4)
    print(f"{len(exporting_roles)} roles successfully exported to {filename}")
```

IIRC the encoding argument is needed for Python 3.6. Also the `dumps` part unnecessarily serialises data to memory.